### PR TITLE
Support for Python 3.9

### PR DESCRIPTION
**Pull request by mistafunk**

### CODE REVIEW

File: envs/centos7/py39/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+sphinxcontrib-serializinghtml==1.1.4
+terminado==0.9.4
+testpath==0.4.4
+tomlkit==0.7.0
+tornado==6.1
+tqdm==4.60.0
+traitlets==5.0.5
+twine==3.4.1
+ujson==4.0.2
+urllib3==1.26.4
```
Comment (clechot) [inline envs/centos7/py39/requirements.txt:104]: Based on the security analysis performed by dependabot, urllib3 needs to be updated to version 1.26.5.

File: envs/windows/py39/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+pycparser==2.20
+Pygments==2.9.0
+pyparsing==2.4.7
+pypiwin32==223
+pyrsistent==0.17.3
+pyshp==2.1.3
+python-certifi-win32==1.6
+python-dateutil==2.8.1
+pytz==2021.1
+pywin32==300
```
Comment (clechot) [inline envs/windows/py39/requirements.txt:77]: Based on dependabot security analysis, we need `pywin32=301`.

File: envs/windows/py39/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+sphinxcontrib-serializinghtml==1.1.4
+terminado==0.9.4
+testpath==0.4.4
+tomlkit==0.7.0
+tornado==6.1
+tqdm==4.60.0
+traitlets==5.0.5
+twine==3.4.1
+ujson==4.0.2
+urllib3==1.26.4
```
Comment (clechot) [inline envs/windows/py39/requirements.txt:110]: Same as for linux, we need `urllib3=1.26.5`.

File: README.md [hunk 4]
```diff
@@ -16,7 +16,9 @@ PyPRT provides a Python binding for PRT (Procedural RunTime) of CityEngine. This
 
 Run `pip install pyprt` in your (virtual) Python environment or `conda install -c esri pyprt` in a Conda environment. Then use `import pyprt` in your scripts.
 
-We provide wheels for Python 3.6 and 3.8 on Linux and Windows. Additionally, we also provide wheels for Python 3.7 on Windows. Conda packages are available for Python 3.6, 3.7 and 3.8 on Linux and Windows. For other Python versions please [build](#development) PyPRT yourself.
+We provide wheels for Python 3.6, 3.8 and 3.9 on Linux and Windows. Additionally, we also provide wheels for Python 3.7 on Windows. Conda packages are available for Python 3.6, 3.7, 3.8 and 3.9(*) on Linux and Windows. For other Python versions please [build](#development) PyPRT yourself.
+
+(*) A note regarding Python 3.9 and Conda: The arcgis package for Python 3.9 is not yet available in Conda, the arcgis submodule in PyPRT is not working yet.
```
Comment (clechot) [inline README.md:21]: > A note regarding Python 3.9 and Conda: The arcgis package for Python 3.9 is not yet available in Conda. The arcgis submodule in PyPRT is **thus** not working yet **for this Python version**.

Or something like that.

**Review by clechot — CHANGES_REQUESTED** [review]

File: envs/centos7/py39/requirements.txt (hunk 1 above)
Comment (mistafunk) [reply envs/centos7/py39/requirements.txt:104]: done

File: envs/windows/py39/requirements.txt (hunk 2 above)
Comment (mistafunk) [reply envs/windows/py39/requirements.txt:77]: done

File: envs/windows/py39/requirements.txt (hunk 3 above)
Comment (mistafunk) [reply envs/windows/py39/requirements.txt:110]: done

File: README.md (hunk 4 above)
Comment (mistafunk) [reply README.md]: done

**Pull request merged**